k8s/k8s.go

```diff
@@ -360,6 +367,15 @@ func (k *k8sOps) SearchNodeByAddresses(addresses []string) (*v1.Node, error) {
 	return nil, fmt.Errorf("failed to find k8s node for given addresses: %v", addresses)
 }
 
+// FindMyNode finds LOCAL Node in Kubernetes cluster.
+func (k *k8sOps) FindMyNode() (*v1.Node, error) {
+	ipList, err := GetLocalIPList(true)
+	if err != nil {
```

Comment:
should we also check if the ipList is not empty here?

Reply:
Don't think it's necessary -- 99.999% of the cases this will be a void check, and `k.SearchNodeByAddresses()` already deals with empty lists.